dll/win32/shell32/CCopyMoveToMenu.cpp

```diff
@@ -334,57 +279,56 @@ CMoveToMenu::QueryContextMenu(HMENU hMenu,
                               UINT idCmdLast,
                               UINT uFlags)
 {
-    MENUITEMINFOW mii;
-    UINT Count = 0;
-
     TRACE("CMoveToMenu::QueryContextMenu(%p, %u, %u, %u, %u)\n",
           hMenu, indexMenu, idCmdFirst, idCmdLast, uFlags);
+    return QueryContextMenuImp(FALSE, hMenu, indexMenu, idCmdFirst, idCmdLast, uFlags);
+}
 
+STDMETHODIMP
+CCopyMoveToMenu::QueryContextMenuImp(BOOL IsCopyOp, HMENU hMenu, UINT indexMenu, UINT idCmdFirst, UINT idCmdLast, UINT uFlags)
```

Comment:
Note that the most commonly seen function name suffix for "implementation" is `Impl`.